<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>702</width>
    <height>683</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Listado de Equipos Electricos</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>20</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>No. Inventario</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>100</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Nombre Objeto</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>150</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Nombre del Local</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>190</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Responsable</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>30</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Consumo</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>80</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Voltaje</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>130</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Marca</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>180</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Modelo</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="table">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>290</y>
     <width>691</width>
     <height>311</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>691</width>
     <height>0</height>
    </size>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::SingleSelection</enum>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
   <property name="sortingEnabled">
    <bool>false</bool>
   </property>
   <column>
    <property name="text">
     <string>No. Inventario</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Fecha</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Nombre de Objeto</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Nombre del Local</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Responsable</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Consumo</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Voltaje</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Marca</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Modelo</string>
    </property>
   </column>
  </widget>
  <widget class="QSpinBox" name="spin_inventario">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>20</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>999999999</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spin_consumo">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>30</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>999999999</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_nom_objeto">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>100</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_nom_local">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>150</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="combo_responsable">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>190</y>
     <width>171</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="combo_voltaje">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>80</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>110</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>220</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="btn_insertar">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>610</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Insertar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_modificar">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>610</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Actualizar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_eliminar">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>610</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Eliminar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_exportar">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>610</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Importar</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>60</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Fecha</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="fecha">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>60</y>
     <width>110</width>
     <height>22</height>
    </rect>
   </property>
   <property name="displayFormat">
    <string>d-M-yyyy</string>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_marca">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>130</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_modelo">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>180</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_buscar">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>240</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>240</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Buscar</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
